examples/log-attack1/index.rsh

```diff
@@ -0,0 +1,30 @@
+'reach 0.1';
+
+export const main = Reach.App(() => {
+  const Alice = Participant('Alice', {
+    Inc: Fun([], UInt),
+  });
+  deploy();
+
+  Alice.publish();
+
+  var [ i ] = [ 0 ];
```

Comment:
You should simplify this so there is just two publishes and the first one gives the consensus access to the contract address and then calls it. The second is just there to block.